docs/index.md

```diff
@@ -2,6 +2,10 @@
 
 Victory is an opinionated, but fully overridable, ecosystem of composable React components for building interactive data visualizations. The following tutorial should help you get started with Victory. For more advanced examples, check out [our guides].
 
+### Getting Started with Victory Native? 
+
+[Check out the native version of this getting started tutorial]
```

Comment:
wait, I thought markdown links with the link at the bottom needed to be in this format: 
```markdown
[link][]

[link]: http://sweet-link.com
```

am I crazy?